src/main/java/org/codice/itest/result/FailureTestResultImpl.java

```diff
@@ -76,4 +81,6 @@ public int hashCode() {
     public String getFailMessage() {
         return this.failMessage;
     }
+
+    private Object getStackTrace() { return stackTrace; }
```

Comment:
❓ Probably won't make much of a difference, but why is the return type `Object` instead of `String`?

Reply:
Good catch, IntelliJ auto generated it and I forgot to change it.